backend/databasemodel/model.sql

```diff
@@ -1470,9 +1482,12 @@ CREATE TABLE kooste.tamperewfs_luontopolkurastit (
 	rasti integer,
 	"infoFi" text,
 	lisatietoja text,
+	deleted boolean NOT NULL DEFAULT false,
 	CONSTRAINT tamperewfs_luontopolkurastit_pk PRIMARY KEY (mi_prinx),
 	UNIQUE (tunnus, rasti)
 );
+CREATE INDEX ON kooste.tamperewfs_luontopolkurastit (deleted);
+CREATE INDEX ON kooste.tamperewfs_luonnonmuistomerkit (visibility);
```

Comment:
I think there is wrong table name here (kooste.tamperewfs_luonnonmuistomerkit -> kooste.tamperewfs_luontopolkurastit) :)